### PR TITLE
Docs update

### PR DESCRIPTION
**Pull request by sakthi-vetrivel**

### CODE REVIEW

File: docs/howto/contents.md [hunk 1]
```diff
@@ -8,7 +8,11 @@
 2. [Developing](development.md)
     Find useful information for running the operator locally.
 
+<<<<<<< HEAD
```
Comment (ultimateboy) [inline docs/howto/contents.md:11]: Merge conflict.

File: README.md [hunk 2]
```diff
@@ -39,9 +39,9 @@ For more details on the control flow of the Azure Service operator, refer to [Az
 
 ![Deploying ASO](/docs/images/asodeploy.gif)
 
-Do you want to quickly deploy the latest version of Azure Service Operator on your Kubernetes cluster and start exploring? Follow these steps.
+Ready to quickly deploy the latest version of Azure Service Operator on your Kubernetes cluster and start exploring? Follow these steps.
 
-1. Make sure `kubectl` is configured to connect to the Kubernetes cluster you want to deploy Azure Service Operators to.
+1. Make sure `kubectl` is configured to connect to the Kubernetes cluster you want to deploy Azure Service Operator to.
```
Comment (theunrepentantgeek) [inline README.md:44]: Perhaps 

```suggestion
1. Make sure `kubectl` is configured to connect to the Kubernetes cluster you want to deploy Azure Service Operator into.
```

File: docs/design/controlflow.md [hunk 3]
```diff
@@ -1,47 +1,41 @@
-## Deploying the operator - How it works
+## Deploying the operator - how it works
 
-![](/docs/images/DeployFlow.png)
+![](/docs/images/Deploy%20Flow.png)
 
 1. The user provisions the Managed Identity (or service principal) in their environment for the Azure Service Operator to use.
+2. The user then deploy cert-manager for the Azure Service Operator to use when deployed.
```
Comment (theunrepentantgeek) [inline docs/design/controlflow.md:6]:
```suggestion
2. The user then deploys cert-manager for the Azure Service Operator to use when deployed.
```

File: docs/design/controlflow.md [hunk 4]
```diff
@@ -1,47 +1,41 @@
...
-
-5. The Azure Service Operator requests an authorizer from Azure Active Directory for the Azure resource management endpoint, as the identity it is running as.
-
-6. The Azure Service Operator then sends the provisioning request to Azure API, along with the authorizer in the request.
+6. The Azure Service Operator then sends the provisioning request to Azure API, along with the authorizer token in the request.
 
 7. Azure API provisions/deprovisions the resource and returns the Resource object to the Service Operator.
 
-8. The Azure Service Operator retrieves the information required to access/consume the Azure resource from the Resource objecrt and stores it in a Kubernetes secret or as a secret in a specified Azure KeyVault.
+8. The Azure Service Operator retrieves the information required to access/consume the Azure resource from the Resource object and stores it in a Kubernetes secret or as a secret in a pre-specified Azure KeyVault.
```
Comment (theunrepentantgeek) [inline docs/design/controlflow.md:29]:
```suggestion
8. The Azure Service Operator retrieves the information required to access/consume the Azure resource from the Resource object and stores it in a Kubernetes secret, or as a secret in a pre-specified Azure KeyVault.
```

File: docs/faq.md [hunk 5]
```diff
@@ -1,6 +1,6 @@
 # Azure Service Operator FAQ
 
-This page provides help with the most common questions about Azure Service Operators
+This page provides help with the most common questions about Azure Service Operators.
 
 ### Do I have to order the creation of resources through the operator? For instance, do I need to first create the EventHub namespace before creating the EventHub?
 No, you can issue the creation of all resources at the same time. The operator will take care of waiting and requeing the requests until the parent resource is ready and all resources will eventually be ready.
```
Comment (theunrepentantgeek) [inline docs/faq.md:6]:
```suggestion
No, you can issue the creation of all resources at the same time. The operator will take care of waiting and requeuing the requests until the parent resource is ready and all resources will eventually be ready.
```

File: docs/faq.md [hunk 6]
```diff
@@ -9,7 +9,7 @@ No, you can issue the creation of all resources at the same time. The operator w
 Yes, we follow semantic versioning to preserve backward compatibility. Please follow Kubebuilder's versioning [mechanism](https://book.kubebuilder.io/multiversion-tutorial/conversion-concepts.html) for this.
 
 ### Is there a guide that talks about how to develop an operator for a new Azure service?
-There is a step-by-step guide that walks you through this process [here](/howto/newoperatorguide.md)
+There is a step-by-step guide that walks you through this process [here](/howto/newoperatorguide.md).
```
Comment (theunrepentantgeek) [inline docs/faq.md:12]:
```suggestion
There is a [step-by-step guide](/howto/newoperatorguide.md) that walks you through this process.
```

File: README.md (hunk 2 above)
Comment (frodopwns) [reply README.md:44]: I think both of these are correct

File: docs/howto/contents.md (hunk 1 above)
Comment (frodopwns) [inline docs/howto/contents.md:11]:
```suggestion
```

File: docs/howto/contents.md [hunk 7]
```diff
@@ -8,7 +8,10 @@
 2. [Developing](development.md)
     Find useful information for running the operator locally.
 
+3. [Testing](test.md)
```
Comment (frodopwns) [inline docs/howto/contents.md:11]:
```suggestion
```

File: docs/howto/contents.md [hunk 8]
```diff
@@ -8,7 +8,9 @@
 2. [Developing](development.md)
     Find useful information for running the operator locally.
 
+=======
```
Comment (frodopwns) [inline docs/howto/contents.md:11]:
```suggestion
```

File: docs/howto/contents.md [hunk 9]
```diff
@@ -9,6 +9,7 @@
     Find useful information for running the operator locally.
 
 3. [Testing](testing.md)
+>>>>>>> 0ddc3c41beee3638a0e0ddc27544788798d18cbf
```
Comment (frodopwns) [inline docs/howto/contents.md:12]:
```suggestion
```

**Pull request merged**